pkg/steps/multi_stage/run.go

```diff
@@ -123,7 +127,7 @@ func (s *multiStageTestStep) runObservers(ctx, textCtx context.Context, pods []c
 	for _, pod := range pods {
 		go func(p coreapi.Pod) {
 			<-ctx.Done()
-			logrus.Info("Signalling observers to terminate...")
+			logrus.Info(TerminateObserversLog)
```

Comment:
Why did you change this to a constant?

Reply:
Because it is used in the `e2e` test code as well